setup.sh

```diff
@@ -108,26 +131,27 @@ function confirmChoices() {
 
 function installAnew() {
     local projects
-    projects=($*)
-    for project in $projects; do
+    projects=("$@")
+    for project in "${projects[@]}" ; do
         (
```

Comment:
No need for a subshell now that you got rid of `cd`.

Reply:
Subshelling was done by this script to be able to install in parallel